config.go

```diff
@@ -141,6 +141,15 @@ func getConfig() *types.Configuration {
 	v.SetDefault("Kubeless.Port", 8080)
 	v.SetDefault("Kubeless.Kubeconfig", "")
 	v.SetDefault("Kubeless.MinimumPriority", "")
+
+	v.SetDefault("Openfaas.GatewayNamespace", "openfaas")
+	v.SetDefault("Openfaas.GatewayAddress", "gateway")
```

Comment:
```suggestion
	v.SetDefault("Openfaas.GatewayService", "gateway")
```
or

```suggestion
	v.SetDefault("Openfaas.GatewayName", "gateway")
```

I think the term `Address` is confusing, we expect something like `http://xxxx` when we see that pattern. In our case, it's a kubernetes service so we only need its name.

My favorite is `GatewayService`, I find it more explicit.

Reply:
yep, GatewayService would be a better name for  the field 👍

Reply:
done.